sympy/ntheory/primetest.py

```diff
@@ -443,6 +443,13 @@ def is_extra_strong_lucas_prp(n):
     #   2) The MathWorld page as of June 2013 specifies Q=-1.  The Lucas
     #      sequence must have Q=1.  See Grantham theorem 2.3, any of the
     #      references on the MathWorld page, or run it and see Q=-1 is wrong.
+
+    """
+    Test cases:
+    V = 0, s= 1, not a pseudoprime
+    V = 0, s= 3, pseudoprime
+    V = 0, s= 5, pseudoprime
+    """
```

Comment:
I think that this docstring can just error out or ignored, so it should be removed.

Reply:
Okay sir